ivy/functional/ivy/device.py

```diff
@@ -689,15 +689,16 @@ def gpu_is_available() -> bool:
 
     Returns
     -------
-    ret
-        Boolean, as to whether a gpu is available.
+    ret: bool
+        Boolean, indicating whether a GPU is available.
```

Comment:
This is just a matter of English semantics, I believe yours flow better so feel free to keep this.